src/hardware/mixer.cpp

```diff
@@ -730,7 +763,7 @@ void MixerChannel::SetReverbLevel(const float level)
 
 float MixerChannel::GetReverbLevel()
 {
-	return reverb.level;
+	return do_reverb_send ? reverb.level : 0.0f;
```

Comment:
No need for this, just return `reverb.level`. If it's off, we set it to 0.0f in the setup. If it's not off, it's set to something else.